web-app-sql-docdb-search/README.md

```diff
@@ -0,0 +1 @@
+# Provision a web app with a SQL Database, DocumentDB and Azure Search

```

Comment:
Readme needs more details about the templates and "deploy to Azure" and visualize buttons